build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1997,6 +2037,16 @@ message Compressor {
 
     // Brotli compression.
     BROTLI = 3;
+
+    // Zstandard compression with dictionary.
+    //
+    // Servers which support this compressor MUST implements
```

Comment:
s/implements/implement/

Reply:
fixed

Reply:
I still see this in the latest version of the PR. Sure you pushed the latest version?

Reply:
Sorry, I pushed it to a dedicated ref on my fork remote. Did not realize this PR was based on the `main` branch of my fork.

I re-pushed the latest version